dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -124,15 +120,22 @@ public String getName() {
 
     @Override
     public String getTitle(){
-    	if(map.get("title") !=null){
-    		return map.get("title").toString();
-    	}
-
     	try {
-    	    ContentletAPI conAPI = APILocator.getContentletAPI();
-    	    String x = conAPI.getName(this, APILocator.getUserAPI().getSystemUser(), false);
-    	    map.put("title", x);
-    	    return x;
+
+    		//Verifies if the content type has defined a title field
+			Optional<com.dotcms.contenttype.model.field.Field> fieldFound = this.getContentType().fields().stream().
+					filter(field -> field.variable().equals("title")).findAny();
+
+
+			if (fieldFound != null && !fieldFound.isPresent()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this null-check of an "Optional". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2789)